pandas/_libs/lib.pyx

```diff
@@ -1955,7 +1957,8 @@ def maybe_convert_numeric(ndarray[object] values, set na_values,
 @cython.wraparound(False)
 def maybe_convert_objects(ndarray[object] objects, bint try_float=0,
                           bint safe=0, bint convert_datetime=0,
-                          bint convert_timedelta=0):
+                          bint convert_timedelta=0,
+                          bint convert_to_nullable_integer=0):
     """
```

Comment:
if you can update this doc-string here e.g. Returns / Parameters

Reply:
Doc-string added